crates/sqlbuiltins/src/functions/scalars/unwind.rs

```diff
@@ -0,0 +1,180 @@
+use bson::Document;
+use datafusion::arrow::datatypes::Fields;
+use serde_json::{from_str, Map, Value};
+
+use super::*;
+use datasources::bson::scalar as bson_scalar;
+
+pub struct Json;
+
+impl ConstBuiltinFunction for Json {
+    const NAME: &'static str = "unwind_json";
```

Comment:
I think grouping the `json` functions under a `json_` prefix improves discovery & autocomplete ux
```suggestion
    const NAME: &'static str = "json_unwind";
```

Reply:
We're pretty inconsistent with this historically: `read_x` vs `scan_x`, etc. I think `read_x` won out anyway (and this is consistent with Duck). 

function names should generally be verb phrases... "do_thing" ... rather than a description of the the outcome.... Functions are named `add` or `add_integers` not `integer_add`.

In many cases autocomplete is a fuzzy search anyway so I think it's fine.